.eslintrc.js

```diff
@@ -190,6 +190,46 @@ module.exports = {
         "@typescript-eslint/prefer-readonly-parameter-types": "off",
       },
     },
+    {
+      files: "*.mdx",
+      extends: ["plugin:mdx/recommended"],
+      rules: {
+        "react/jsx-filename-extension": ["error", { extensions: [".mdx"] }],
+        "mdx/no-unescaped-entities": "OFF",
+        "mdx/remark": "error",
+        "import/extensions": [
+          "error",
+          "ignorePackages",
+          {
+            tsx: "never",
+          },
+        ],
+      },
+      settings: {
+        "import/resolver": {
```

Comment:
[`eslint-import-resolver-typescript`](https://github.com/alexgorbatchev/eslint-import-resolver-typescript) should be preferred for ts files.

Reply:
I agree even our `*.ts?(x)` files don't seem to have the resolver set up, I can't remember what was the problem I ran into… 🤔